# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/library/rust/shared/lib.rs
# Commit: 2b647eb982bc
# Full Hash: 2b647eb982bc0068d39fe58c6f59bebe55909343
# Author: Dana Keeler <dkeeler@mozilla.com>
# Date: 2023-06-07 21:43:58
# Regressor Bug: 1834862
# File Overlap Count: 1
# Description:
#   Bug 1834862 - implement nsICryptoHash in rust r=jschanck,supply-chain-reviewers
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D179727
# ==============================================================================

diff -r 7c11166969ff -r 2b647eb982bc toolkit/library/rust/shared/lib.rs
--- a/toolkit/library/rust/shared/lib.rs	Wed Jun 07 15:45:14 2023 +0000
+++ b/toolkit/library/rust/shared/lib.rs	Wed Jun 07 15:56:37 2023 +0000
@@ -16,6 +16,7 @@
 extern crate bookmark_sync;
 extern crate cascade_bloom_filter;
 extern crate cert_storage;
+extern crate crypto_hash;
 extern crate chardetng_c;
 extern crate cosec;
 #[cfg(feature = "cubeb_coreaudio_rust")]
